docroot/themes/humsci/humsci_basic/src/scss/components/_global-message.scss

```diff
@@ -0,0 +1,125 @@
+// This is a stylesheet for the Global Message banner
+.su-alert {
+  background-color: #f4f4f4;
+  color: #2e2d29;
+  padding: 0;
+
+  & .fa,
+  & .fas,
+  .hb-dark-pattern & .fa,
+  .hb-dark-pattern & .fas {
+    color: #2e2d29;
+  }
+
+  .hb-dark-pattern & a,
+  a {
+    color: #2e2d29;
+    border-bottom: 0;
+    background-image: none;
+    text-decoration: none;
+    font-weight: hb-theme-font-weight(semibold);
+
+    &:hover {
+      color: #2e2d29;
+      text-decoration: underline;
+    }
+  }
+
+  &--plain {
+    background-color: #f4f4f4;
+  }
+
+  &--error {
+    background-color: #b1040e;
+  }
+
+  &--success {
+    background-color: #008566;
+  }
+
+  &--warning {
+    background-color: #eaab00;
+  }
+
+  &--info {
+    background-color: #006cb8;
+  }
+
+  &--text-light,
+  &--text-light a,
+  &--text-light .fa,
+  &--text-light .fas,
+  .hb-dark-pattern &--text-light a,
+  .hb-dark-pattern &--text-light .fa,
+  .hb-dark-pattern &--text-light .fas {
+    @include hb-global-color('color','white');
+
+    &:hover {
+      @include hb-global-color('color','white');
+    }
+  }
+
+  &__header {
+    margin-bottom: 1em;
+    width: 100%;
+    margin-right: hb-calculate-rems(20px);
+
+    @include grid-media-min('lg') {
+      margin-right: hb-calculate-rems(26px);
+      order: 1;
+      flex-shrink: 1;
+      width: auto;
+      margin-bottom: 0;
+    }
+  }
+
+  &__label {
+    height: 100%;
+    line-height: 100%;
+    display: inline-block;
+    font-size: hb-calculate-rems(17px);
+    text-transform: uppercase;
+    font-weight: hb-theme-font-weight(semibold);
+
+    &::after {
+      content: ":";
+    }
+  }
+
+  &__body {
+    order: 2;
+    flex-grow: 1;
+    flex-basis: 100px;
```

Comment:
Can we use `hb-calculate-rems` here?

Reply:
I did go ahead and add rem to this instead, did a little testing and didn't see an impact from using rems instead.